include/seqan3/contrib/charconv/charconv.cpp

```diff
@@ -0,0 +1,235 @@
+//===------------------------- charconv.cpp -------------------------------===//
```

Comment:
What's this file for?

Reply:
https://github.com/llvm/llvm-project/blob/d27489645bb8829a952f79e3b211202844238391/libcxx/src/charconv.cpp

We import the file as is and try to do as little modification as possible.